### ISSUE TITLE
ConfigRouter undefined

### ISSUE THREAD

**Issue by danielthatcher** [nccgroup/tracy#15]
Currently "make test" fails as ConfigRouter is undefined at [config_server_helper_test.go:39](https://github.com/nccgroup/tracy/blob/master/api/rest/config_server_helper_test.go#L39).

```
$ make test
...
api/rest/config_server_helper_test.go:39:3: undefined: ConfigRouter
...
```

Naively grepping suggests that maybe it should be "RestRouter"

**Comment by jacobheath** [comment]
I need to update that test. ConfigRouter was removed in 613c9c45f011bde507053c07b0adcddc5f2147b3. Maybe the test doesn't need to be there anymore, actually.

**Comment by jacobheath** [comment]
Fixed with daace77c08603633017db7b2ee44d0fb25321a11
